Tests/CodegenCLITests/Commands/GenerateTests.swift

```diff
@@ -445,4 +446,106 @@ class GenerateTests: XCTestCase {
     // then
     expect(level).toEventually(equal(.debug))
   }
+
+  // MARK: Version Checking Tests
```

Comment:
Should we have another test here to simulate a CocoaPods environment, i.e.: any other dependency manager where `.noApolloVersionFound` would be returned because there is no package file. I see there's a test for that in the version checker but we have no validation for how the generate command responds in that case.

Reply:
Sure, test added!